code/modules/urist/modules/jungle/jungle_animals.dm

```diff
@@ -43,12 +45,89 @@
 	name = "snake spawner"
 	spawn_type = /mob/living/simple_animal/hostile/snake
 
+/obj/effect/landmark/animal_spawner/random
+	var/spawn_list
+	var/crosstrigger = 0
+	var/x_offset = 0
+	var/y_offset = 0
+
+/obj/effect/landmark/animal_spawner/random/New()
+	if(crosstrigger)
+		return
+
+	else
+		processing_objects.Add(src)
+		spawn_type = pick(spawn_list)
+		spawned_animal = new spawn_type(get_turf(src))
+
+/obj/effect/landmark/animal_spawner/random/process()
+	//if any of our animals are killed, spawn new ones
+	if(!spawned_animal || spawned_animal.stat == DEAD)
+		spawn_type = pick(spawn_list)
+		spawned_animal = new spawn_type(src)
+		//after a random timeout, and in a random position (6-30 seconds)
+		spawn(rand(spawn_time_low,spawn_time_high))
+			spawned_animal.loc = locate(src.x + x_offset, src.y + x_offset, src.z)
+
+/obj/effect/landmark/animal_spawner/random/Crossed(mob/living/simple_animal/M)
+	if(crosstrigger) //if an animal crosses this thing, they "leave" the map, and then this landmark starts spawning animals
+		qdel(M)
+		processing_objects.Add(src)
+		return
+
+	else
+		..()
+
+/obj/effect/landmark/animal_spawner/random/jungle
+	name = "jungle animal spawner"
+	x_offset = -8
+	spawn_list = list(
+		/mob/living/simple_animal/hostile/huntable/panther,
+		/mob/living/carbon/human/monkey,
+		/mob/living/simple_animal/parrot,
+		/mob/living/simple_animal/hostile/huntable/deer
+		)
+
+/obj/effect/landmark/animal_spawner/random/plains
+	name = "plains animal spawner"
+	y_offset = -2
+	spawn_list = list(
+		/mob/living/simple_animal/hostile/huntable/bear,
+		/mob/living/simple_animal/hostile/huntable/deer,
+		/mob/living/simple_animal/hostile/snake
+		)
+
+/obj/effect/landmark/animal_spawner/random/jungle/crosstrigger
+	crosstrigger = 1
+	icon_state = "x2"
+
+/obj/effect/landmark/animal_spawner/random/plains/crosstrigger
+	crosstrigger = 1
+	icon_state = "x2"
+
+//huntable animals
+
+/mob/living/simple_animal/hostile/huntable
+	var/hide = 0
+
+/mob/living/simple_animal/hostile/huntable/attackby(var/obj/item/I, mob/user as mob)
+	if(istype(I, /obj/item/weapon/material/knife) && src.stat == DEAD)
+		if (do_after(user, 60, src))
+			user << "<span class='notice'>You gut and skin [src], getting some usable meat and hide.</span>"
+			new meat_type(src.loc)
+			new meat_type(src.loc)
```

Comment:
var/meat (in huntable)
for(i, i<=meat, i++) 
    new meat_type(src.loc)?
Might as well make that customizable too, so that a bear yields more than a duck.